Dockerfile

```diff
@@ -43,6 +43,12 @@ RUN apt-get update \
 		wget \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
+	wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
+  apt-get update && \
+  apt-get install -y postgresql-client-11 && \
```

Comment:
I think this also needs to go in the no systemd image, cc @roman-mazur?

Reply:
Yes, I will appreciate if this can be added to the no systemd variant.
https://github.com/balena-io/open-balena-base/blob/master/Dockerfile.no-systemd#L6

Reply:
Done!